frontend/src/main/components/BasicCourseSearch/CourseInstructorSearchForm.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+import { quarterRange } from "main/utils/quarterUtilities";
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import { useBackend  } from "main/utils/useBackend";
+
+const CourseInstructorSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localStartQuarter = localStorage.getItem("CourseInstructorSearch.StartQuarter");
+  const localEndQuarter = localStorage.getItem("CourseInstructorSearch.EndQuarter");
+  const localInstructor = localStorage.getItem("CourseInstructorSearch.instructor");
+
+  const {error: _error, status: _status } =
+  useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    ["/api/UCSBSubjects/all"], 
+    { method: "GET", url: "/api/UCSBSubjects/all" }, 
+    []
+  );
+
+  const [startQuarter, setStartQuarter] = useState(localStartQuarter || quarters[0].yyyyq);
+  const [endQuarter, setEndQuarter] = useState(localEndQuarter || quarters[0].yyyyq);
+  const [instructor, setInstructor] = useState(localInstructor || "");
+    
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    fetchJSON(event, { startQuarter, endQuarter, instructor });
+  };
+
+
+  // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
Please remove this, and add a tests for the CSS properties
(See: https://ucsb-cs156.github.io/topics/stryker/stryker_html_attribute_surviving_mutants.html for instructions on how.)

Plus anything else that Stryker flags as needing mutation coverage.

This is leftover from when we didn't know how to do that easily, and even then, it was overly broad, excluding far too much.